packages/@okta/vuepress-site/docs/guides/terraform-manage-user-access/main/index.md

```diff
@@ -140,7 +140,7 @@ Configure the email authenticator and then create a group, global session policy
 1. Add an `okta_policy_mfa` resource to create an authenticator enrollment policy.
 
    1. Set `status` to `ACTIVE`.
-   1. Set `is_oie` to `true` if you’re using the Okta Identity Engine.
+   1. Set `is_oie` to `true`. Okta Classic supports only setting `okta_password` to `REQUIRED`.
    1. Set `groups_included` to the passwordless group ID.
    1. Set `okta_email` to `REQUIRED` to require users to enroll in the email authenticator.
    1. Set `okta_password` to `NOT_ALLOWED` to prevent users from enrolling in the password authenticator.
```

Comment:
Match any changes to the below content as well.

Reply:
FYI: I worked with Jeff on the text and your suggestion is the same as what I originally suggested. However, Jeff wanted to place the information at the point of use rather than as a note afterwards.

Reply:
There are 3 issues:
- setting `okta_password` to `REQUIRED` breaks the examples
- the explanation is removed from the point of correction (only setting `is_oie` to `true`)
- the number of customers still on classic means that some readers will be on classic and we don't have the contract dollars to either fix the scripts or add class-only versions.

What makes the inline placement confusing? Is it the wording?

Reply:
The wording itself is fine _I guess_ (I'd still probably change the wording to `Classic orgs must set okta_password to REQUIRED`), but the placement is confusing and doesn't provide anything to readers.

* The step explains how to configure the type of org. but we're removing the context on what the `is_oie` setting does. The current wording is mostly OK and keeps the reader focused on what they need to know.
  * We should also adjust the wording to provide context for Classic orgs. 
  * Do they just just not set the param? Do they set it to `false` or `null`?
  * What if we changed it to something like `1. For OIE orgs, set `is_oie` to `true`. For Classic orgs, <whatever>.`
* We're mentioning `okta_password ` in the same step as another setting, and not connecting things for customers. Sure, they know they need can only set `okta_password` to required, but does it even matter to them? It's three more steps before we mention that setting again.
* We still mention `okta_password` a few steps later with a different option. There's no added context on step 5 that explains that `NOT_ALLOWED` isn't available for Classic orgs and it's been three steps since we mentioned the note. If they're chunked together, the user can easily see, "Oh, I'm on a Classic org, I need to change this to REQUIRED!"

Also, if the examples are for an OIE org, mention that upfront.
> This guide was created for OIE orgs. While it may work for Classic orgs, you may need to adjust some settings.

Reply:
I don't know any Terraform specific answers. All of the material was created by a contract writer and we ran out of dollars (at least for now.) I also don't know if it's semantically accurate to say that everything is written for OIE... it may be that just those two scripts are and nothing else, but I have no way of knowing that. In other words, saying that the guide was created for OIE may be semantically inaccurate. It's also not the nicest thing to tell about 60% of the customers using Terraform that they may need to adjust some settings.

I agree that the proper fix is to provide information on what to do for Classic, or even to provide Classic code alongside OIE... but we don't have the people to do that.

I went for the fastest and most localized fix, which is to let folks who use Classic know that they can only have a password setting of required. Then it's up to them to figure out what needs doing.

Reply:
I don't see any reason to not include that content inline. But, again, I would say Okta Classic Engine.